com.github.bcedu.valasimplehttpserver.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id": "com.github.bcedu.valasimplehttpserver",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.bcedu.valasimplehttpserver",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro"
+    ],
+    "build-options": {
```

Comment:
Those build-options are now used by default with FreeDesktop runtime 18.08 and GNOME 3.30. You can remove them.